hypha/apply/funds/models/applications.py

```diff
@@ -343,9 +344,73 @@ def get_form_parameters(self, submission_id=None):
+                    field_from_block.grouper_for = group_counter + 1
+                    group_counter += 1
+                    is_in_group = True
+                    sub_children_visible = posted_field_values.get(struct_child.id) == field_from_block.choices[0][0]
+                if isinstance(block, TextFieldBlock):
+                    field_from_block.word_limit = struct_value.get('word_limit')
+                if isinstance(block, MultiInputCharFieldBlock):
+                    number_of_inputs = struct_value.get('number_of_inputs')
+                    for index in range(number_of_inputs):
+                        form_fields[struct_child.id + '_' + str(index)] = field_from_block
+                        field_from_block.multi_input_id = struct_child.id
+                        field_from_block.add_button_text = struct_value.get('add_button_text')
+                        if index == number_of_inputs - 1:  # Add button after last input field
+                            field_from_block.multi_input_add_button = True
+                            # Index for field until which fields will be visible to applicant.
+                            # Initially only the first field with id UUID_0 will be visible.
+                            field_from_block.visibility_index = 0
+                            field_from_block.max_index = index
+                        if index != 0:
+                            field_from_block.multi_input_field = True
+                            field_from_block.required = False
+                            field_from_block.initial = None
+                        field_from_block = copy.copy(field_from_block)
+                else:
+                    if is_in_group:
+                        field_from_block.required_when_visible = field_from_block.required
+                        field_from_block.required = field_from_block.required & sub_children_visible
+                        field_from_block.visible = sub_children_visible
+                    form_fields[struct_child.id] = field_from_block
+            elif isinstance(block, GroupToggleEndBlock):
+                # Group toogle end block is used only to group fields and not used in actual form.
+                # Todo: Use streamblock to create nested form field blocks, a more elegant method to group form fields.
+                is_in_group = False
+            else:
+                field_wrapper = BlockFieldWrapper(struct_child)
+                field_wrapper.group_number = group_counter if is_in_group else 1
+                form_fields[struct_child.id] = field_wrapper
+
+        return form_fields
+
+    def get_form_class(self, draft=False, posted_field_values={}):
+        return type('WagtailStreamForm', (self.submission_form_class,), self.get_form_fields(draft, posted_field_values))
+
     def get_form(self, *args, **kwargs):
         draft = kwargs.pop('draft', False)
-        form_class = self.get_form_class(draft)
+        from_class = None
```

Comment:
Why do we need to assign form_class as None?

Reply:
Removed. Isn't necessary.